CODEOWNERS

```diff
@@ -217,6 +217,7 @@ Kconfig*                                  @tejlmand
 /tests/lib/hw_unique_key*/                @oyvindronningstad @Vge0rge
 /tests/lib/location/                      @trantanen @hiltunent @jhirsi @tokangas
 /tests/lib/lte_lc/                        @jtguggedal @tokangas @simensrostad
+/tests/lib/lte_lc_api/                    @maxd-nordic
```

Comment:
Is this supposed to be part of this PR?

Reply:
I guess yes if we keep lte_lc_api test commit in this PR.

Reply:
@balaji-nordic Was there a reason to remove this change from this PR?

Reply:
Not anymore. Initially I thought to keep this PR clean and remove unrelated changes. But since this PR goes in as three separate commits, this is no longer a concern for me.